app/javascript/components/UserAssignmentForm.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import UserAssignmentFormField from "./UserAssignmentFormField";
+
+class UserAssignmentForm extends React.Component {
+  constructor(props) {
+    super();
+    this.state = props;
+    this.handleAddClick = this.handleAddClick.bind(this);
+    this.handleDeleteClick = this.handleDeleteClick.bind(this);
+  }
+
+  randomUUID() {
+    return "new-mission-"+ Math.floor(Math.random() * Math.floor(9000));
```

Comment:
Space before `+`. Bummer that linter didn't catch :(

Reply:
`Math.floor(9000)` is the same as just `9000`. If you copied this from somewhere please fix there too.